Dangerfile

```diff
@@ -17,3 +17,7 @@ end
 if pr_body.length < 5
   fail "Please provide a changelog summary in the Pull Request description @#{pr_author}"
 end
+
+unless files_modified.any? { |a| a.include?("CHANGELOG.md") }
```

Comment:
@KrauseFx would it feel reasonable to allow something like this using custom objects that act array-ish but provide dictionary-like DSL attribtues?

``` ruby
warn("Please include a CHANGELOG entry") unless file_modified["CHANGELOG.md"]
```
```